### PR TITLE
Implement game_play event

### PR DESCRIPTION
**Pull request by kpman**
clsoe #262

### CODE REVIEW

**Comment by tw0517tw** [comment]
As `payload` is JSON encoded data, would it be better if we do `JSON.parse()` here?

**Comment by kpman** [comment]
Good catch!
Already implemented it on this PR.

File: src/context/MessengerEvent.js [hunk 1]
```diff
@@ -77,6 +77,15 @@ export type Postback = {
   referral?: Referral,
 };
 
+export type GamePlay = {
+  game_id: string,
+  player_id: string,
+  context_type: 'SOLO' | 'THREAD' | 'GROUP',
+  context_id: string,
+  score: number,
+  payload: string,
```
Comment (chentsulin) [inline src/context/MessengerEvent.js]: Object?

Comment (tw0517tw) [reply src/context/MessengerEvent.js]: It's JSON string in the rawEvent, but we convert it to JS object in event.gamePlay getter.

**Pull request merged**